Makefile

```diff
@@ -138,17 +138,24 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+DEPLOY_TMP=/tmp/oadp-make-deploy
+deploy-tmp: kustomize
```

Comment:
Does this need to clean up the tmp dir after it runs?

Reply:
Sure thing.

Reply:
It can.. but only in its dependent make targets.

Reply:
resolved in https://github.com/openshift/oadp-operator/pull/506/commits/d38c98934ec7f6402e5bb903fb24511621c19830